pom.xml

```diff
@@ -106,6 +106,9 @@
 		<jaeger.version>1.6.0</jaeger.version>
 		<opentracing.version>0.33.0</opentracing.version>
 		<opentracing-kafka-client.version>0.1.15</opentracing-kafka-client.version>
+		<opentelemetry.version>1.7.0-alpha</opentelemetry.version>
+		<opentelemetry-stable.version>1.7.0</opentelemetry-stable.version>
```

Comment:
why there are two opentelemetry version properties (one with `-stable` and the other seems to be an alpha) using both in different places to import dependencies?

Reply:
"My" OpenTelemetry artifacts did yet made it to "stable" version, where Jaeger artifcat alfready did.
Hence the 2 diff versions.
But we agreed it's OK to use alpha for those -- as they are the initial ones, and haven't been changed.
I'll check if there is a stable version, but shouldn't make a diff.
(if I change it again now, I need to re-run the whole cycle, for the N-th time ...)

Reply:
Did you check if there is a stable version? This "alpha" here bothers me a little bit :-)

Reply:
Let me check ...

Reply:
https://search.maven.org/artifact/io.opentelemetry.instrumentation/opentelemetry-kafka-clients-common

Reply:
All alphas ... but that's OK, since, afaik / imho, the 1st didn't change at all ...